### ISSUE TITLE
RGBA color doesn't work correct with border

### ISSUE THREAD

**Issue by duyank** [diegomura/react-pdf#1491]
**Describe the bug**
When I use border color is `rgba(60, 217, 48, 0.49);`
then it shows like the image below
![image](https://user-images.githubusercontent.com/19285404/132487013-5bad4208-090a-49a8-af2b-2dda12a5ce03.png)

**Expected behavior**
I wish it can show like text

**Comment by tom2strobl** [comment]
I can confirm that rgba values with full opacity (1) work and as soon as transparency is introduced I get a color that makes absolutely no sense, either a full black or a dark blue or whatever, it seems that as soon as alpha is in play some calculation is wildly off.

**Comment by David245M** [comment]
Yep, this issue still exists, same rgba value with alpha is less than 1 and looks different for `borderColor` and `backgroundColor`. 
Reproduction sandbox: https://codesandbox.io/p/devbox/ymm5dc?file=%2Fsrc%2FApp.jsx%3A15%2C31

The temporary solution I have right now is to separate the component into the wrapper and inner components, where the wrapper will have rgb color without alpha value, but the alpha value is set using the `opacity` property. In the inner component, I reset the opacity back to 1 to avoid applying it anywhere else besides the border color. It's not perfect, but it works for me. You can see an example in the same sandbox above.
